.github/workflows/build.yml

```diff
@@ -19,8 +19,7 @@ jobs:
     steps:
       - uses: cisagov/setup-env-github-action@develop
       - uses: actions/checkout@v2
-      - name: Set up Python
-        uses: actions/setup-python@v1
+      - uses: actions/setup-python@v2
```

Comment:
I just noticed that we got rid of `- name: Set up Python` in the `lint` job, but not in the `test` job.  Those two should probably be in sync.  I realize this change belongs in `skeleton-ansible-role` - which way should we go?

1. Remove `- name: Set up Python` in `test` job
1. Add `- name: Set up Python` back in `lint` job

I'll create an issue in [`skeleton-ansible-role`](https://github.com/cisagov/skeleton-ansible-role) once we reach consensus.

Reply:
I'm fine either way, but probably a little more in favor of removing the `name`.

Reply:
I believe it was removed in the lint job because actions/setup-go doesn't have a descriptor either. So if it were to be added back, then we should add a name for every uses. I'm definitely more in favor of just removing the name since the action is self descriptive.

Reply:
I created https://github.com/cisagov/skeleton-ansible-role/pull/53 to address this.